eq-author/src/App/page/Design/answers/MultipleChoiceAnswer/__snapshots__/MultipleChoiceAnswer.test.js.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`MultipleChoiceAnswer exclusive options should render exclusive 1`] = `
-<Connect(withEntityEditor(StatelessBasicAnswer))
+<_class
```

Comment:
I presume this is unintentional

Reply:
yup, I hadn't defined the displayName of withValidationError HoC. Now fixed and snapshots updated